### ISSUE TITLE
Dropdown box not working on VPN Server page

### ISSUE THREAD

**Issue by alfky12** [SWRT-dev/asuswrt-bcm#3]
Router: ASUS TUF-AX3000 (V1)
Firmware version: 3.0.0.4_TUFAX3000_R5.2.4_20126-g1d747fa_cferom_puresqubi

Issue:
On VPN Server page, if we want to select VPN Details, dropdown box not working when clicked, so we cannot select General or Advanced Settings.
Not only VPN Details box, but all dropdown box on all Server List on this VPN Server page.

Screenshot:

![Screenshot (146)](https://github.com/SWRT-dev/asuswrt-bcm/assets/100685677/52c8a23d-333a-4ff7-bebc-80c01351c39f)

**Comment by paldier** [comment]
![20231124120942](https://github.com/SWRT-dev/asuswrt-bcm/assets/2846036/f268cdf4-b595-41b6-89a1-3357f35ffc8f)
No problems found.

**Comment by alfky12** [comment]
Okay, after you said there is no problem from your side, I tried again on Chrome, Edge, Firefox, Opera and Brave browsers.
Only Brave can do it.

Then I realized that on my Chrome, Edge, Firefox and Opera there was an active Adguard addon, on Brave there wasn't.
So I tried turning off the adguard addon, and it worked.
The active Adguard addon is the cause of the problem.

I will close this issue, thanks for the fast response.
